content/guides/tutorials/toc.md

```diff
@@ -238,10 +238,6 @@
 ## [DNN Security](xref:dnn-security)
 ## [More Resources](xref:more-resources)
 
-<!-- START OF "GLOSSSARY" NAV ITEM -->
-# [Glossary](xref:admin-glossary)
```

Comment:
This should be `xref:tutorials-glossary` 
I also think we should move the Glossary entries in the TOC's down toward the bottom, though this can be done at a later date.

Reply:
There is already a link to the correct tutorials link already exists in the bottom and points to the correct `xref`.

Reply:
My bad. good catch.